src/components/Letter.js

```diff
@@ -0,0 +1,14 @@
+import React, { Component } from 'react';
+
+class Letter extends Component {
+    selectLetter = () => {
```

Comment:
can make it one line

`selectLetter = () =>this.props.selectLetter(this.props.letter)`